ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -254,29 +274,29 @@ initLedgerDB replayTracer tracer OldLedgerInitParams{..} NewLedgerInitParams{..}
           let
             ledgerDb = combine Nothing newStyleDB
           in
-            (Nothing,) <$> initStartingWith replayTracer' cfg backend streamAPI ledgerDb
+            (\(l, r) -> (Nothing, (l, (r,r)))) <$> initStartingWith replayTracer' cfg backend streamAPI ledgerDb
```

Comment:
Maybe we want to define a function with a descriptive name that explains what ` (\(l, r) -> (Nothing, (l, (r,r))))` does.

I also wonder if we'd want to introduce a data structure for `(l, (r, r))`